### PR TITLE
Done

### PR DESCRIPTION
**Pull request by Green-stripes**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -22,6 +22,18 @@
           <h2>We're a group of people on a journey to learn to love code.</h2>
         </div>
       </div>
+      <div class="roster-cards-container">
```
Comment (bhollan) [inline index.html]: This line should not be here.  There should only be one `roster-cards-container`.  If you remove this, we should be able to merge this in.

**Comment by bhollan** [comment]
Hi @Green-stripes !  These changes look pretty good except for one extra unwanted line in `index.html`.  I've made a comment on the line to be deleted.  Feel free to Ask A Question if you need any help!

CC: @aturkewi

File: index.html [hunk 2]
```diff
@@ -22,6 +22,18 @@
...
+        <div class="student-card" id="kaveri-bhattacharya-card">
```
Comment (aturkewi) [inline index.html]: This entire student card should also be _inside_ of the existing `roster-cards-container` div below.

**Comment by bhollan** [comment]
Hi @Green-stripes !

I'm going to close this PR for now.  We can re-open it any time you like.  Just let us know whenever you're ready to implement the necessary changes.

If you need any help, as always, please feel free to Ask A Question!

CC: @aturkewi

**Pull request closed without merging**